src/frontend/js/modules/i18n/i18n-loader.service.js

```diff
@@ -23,7 +25,9 @@
       };
 
       function getCatalogs() {
-        return $http.get('/api/i18n').then(function(res) {
+        const url = httpConfigurer.getUrl('/api/i18n');
+
+        return $http.get(url).then(function(res) {
```

Comment:
You should add some comments to remind yourself or others to write tests for this part of the code later. e.g.:

```
// TODO: Write tests for this (https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/pull/79)
```

Reply:
It's done. Please check again